source/documentation/process_flows.html.md

```diff
@@ -0,0 +1,476 @@
+1. Following the result of the risk analysis engine, the office of departure may select the pre-lodged declaration for potential control of the goods prior to their presentation. The following control decisions are possible: 
+    - **Yes** (apply control): the office of departure notifies the holder of the transit procedure (provided that they are an Authorised Economic Operator (AEO)) about the intention to potentially control the goods, via the ‘Control Decision Notification’ E_CTR_DEC (IE060) message (having the data element TRANSIT OPERATION-Notification type equal to ‘2-Intention to Control’). Go to step 4.
+    - **No** (no control): Go to step 4.
+1. The office of departure receives a valid ‘Presentation Notification for the Pre-Lodged Declaration’ E_PRE_NOT (IE170) message from the holder of the transit procedure.
+1. The MRN is communicated to the holder of the transit procedure with message ‘MRN Allocated’ E_MRN_ALL (IE028).
+1. The ‘Release for Transit’ E_REL_TRA (IE029) message is sent to the holder of the transit procedure.
+1. The departure process ends.
+
+
+
+### Transit presentation notification not valid 
+
+**Applicable procedures:** normal and simplified.
+
+This scenario involves the submission of an invalid transit declaration for goods that have not yet been presented to the office of departure.
+
+<img src="../figures/trans_pres_notif_not_valid.svg" alt="Pre-lodgement message flow with presentation of a non-valid notification. Flow is described in this section." />
+
+<a href="../figures/trans_pres_notif_not_valid.svg" target="_blank">Open the diagram in a new tab.</a>
+
+1. The process starts when the holder of the transit procedure submits the ‘Declaration Data’ E_DEC_DAT (IE015) message to the office of departure with ‘Additional Declaration Type’ equal to ‘D’.
+1. The office of departure validates this message successfully and sends the ‘Positive Acknowledgement’ E_POS_ACK (IE928) message to the holder of the transit procedure to acknowledge receipt of the transit declaration.
+1. The office of departure receives an invalid ‘Presentation Notification for the Pre-Lodged Declaration’ E_PRE_NOT (IE170) message from the holder of the transit procedure.
+1. The office of departure decides to reject the E_PRE_NOT (IE170) message. 
+1. The office of departure notifies the holder of the transit procedure with the ‘Rejection from Office of Departure’ E_DEP_REJ (IE056) message.
+1. If the time limit expires without the holder of the transit procedure resending a valid ‘Presentation Notification for the Pre-Lodged Declaration’ E_PRE_NOT (IE170) message, the holder of the transit procedure is notified with the message ‘Rejection from Office of Departure’ E_DEP_REJ (IE056) by the office of departure.
+1. The transit movement ends.
```

Comment:
1, 1, 1, 1,? Surely this should be 1,2,3 etc?

Reply:
You'd have thought so but GitHub markdown knows what to do with it. Advantage: don't have to fiddle about manually renumbering steps whenever you insert a new step 6. Disadvantage: harder to figure out which step to instruct users to jump to for branches. It does work though, and it's documented somewhere as legit.

Reply:
Discussed, it renders correctly locally, happy to merge